common/model-views.h

```diff
@@ -551,10 +552,10 @@ namespace rs2
 
         std::atomic<bool> depth_stream_active;
 
+        const size_t resulting_queue_max_size;
```

Comment:
As I noted outside the review: see [C.47](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rc-order) and [C.48](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rc-in-class-initializer).

Following those rules you'll get:
```cpp
const size_t resulting_queue_max_size = 20;
rs2::frame_queue syncer_queue;
rs2::frame_queue resulting_queue{ resulting_queue_max_size };
```
Which makes it clear and it is all located in the same place.
Also you can see from this that `syncer_queue` is not initialized to any specific value...

Reply:
I would prefer not to initialize the const in the class declaration for two reasons:
- Concentrating all hard-coded values in one place makes it more error-resilient as it is easier to spot cross-referenced values both with debug and peer review.
- The compiler went silent for the current code with the bug. Had the assignment occurred in the initialization list, then the compiler would be able to recognize and report this in real time

Reply:
While the CPP Core Guideline is not the bible, many other people have already had these arguments and have arrived to the conclusion of what is better, so I suggest we get along.